docs/day-2-operations/Backups.md

```diff
@@ -0,0 +1,272 @@
+
+### Persistent Volumes
+In a 3scale deployment on OpenShift all persistent data is stored either in a storage service in the cluster (not currently used), a Persistent Volume (PV) provided to the cluster by the underlying infrastructure or a storage service external to the cluster (be that in the same Data Center or elsewhere).
+
+### Considerations
+The backup and restore procedures for persistent data are required to vary depending on the storage used, to ensure the backups and restores preserve data conistsency (e.g. that a partial write, or a partial transaction is not captured). i.e. It is not sufficient to backup the underlying Persistent Volumes for a database, but instead the databases backup mechanisms should be used.
+
+Also, some parts of the data are synchronized between different components. One copy is considered the "source of truth" for the data set, and the other a copy that is not modified locally, just synchronized from the "source of truth". In these cases, upon restore, the "source of truth" should be restored and then the copies in other components synchronized from it.
+
+## Data Sets
+This section goes into more detail on the different data sets in the different persistent stores, their purposes, the storage type used and if it is the "source of truth" or not.
+
+The full state of a 3scale deployment is stored across these services and their persistent volumes:
+
+* `system-mysql`:  Mysql database (`mysql-storage`)
+* `system-storage`: Volume for Files
+* `zync-database`: Postgres database for `zync` component. This uses "HostPath" as storage. If the pod is moved into another node the data is lost. That is not a problem because the data are sync jobs and does not need to be 100% persistent.
+* `backend-redis`: Redis database (`backend-redis-storage`)
+* `system-redis`: Redis database (`system-redis-storage`)
+
+### `system-mysql`
+This is a relational database for storing the information about users, accounts, APIs, plans etc in the 3scale Admin Console. 
+
+A subset of this information related to Services is synchronized to the `Backend` component and stored in `backend-redis`. `system-mysql` is the source of truth for this information. 
+
+### `system-storage`
+This is for storing files to be read and written by the `System` component. They fall into two categories:
+
+* Configuration files read by the System component at run-time
+* Static files (HTML, CSS, JS, etc) uploaded to System by it's CMS feature, for the purpose of creating a Developer Portal.
+
+Note that `System` can be scaled horizontally with multiple pods uploading and reading said static files, hence the need for a `RWX` PersistentVolume.
+
+### `zync-database`
+This is a relational database for storing information related to the synchronization of identities between 3scale and an Identity provider.
+This information is not duplicated in other components and this is the sole source of truth.
```

Comment:
That's correct. Though `system-database` has some events logging table which could be used to synchronize zync events.

Reply:
OK, related to above comment I think. But that maybe useful information to add in this description somewhere....so people know.

Reply:
@hallelujah could you provide some snippet on how to describe this?